ui/component/livestreamComments/view.jsx

```diff
@@ -38,15 +40,16 @@ export default function LivestreamComments(props: Props) {
     embed,
     doCommentSocketConnect,
     doCommentSocketDisconnect,
-    comments,
+    comments, // superchats in chronological format
     doCommentList,
     fetchingComments,
     doSuperChatList,
-    superChats,
-    superChatsTotalAmount,
     myChannels,
+    superChats, // superchats organized by tip amount
```

Comment:
One thing you can do is get
{ superChats: superchatsSortedByAmount, comments: superchatsSortedByDate } = props; to use a new name after destructuring.

Reply:
Perhaps whatever is connect()ed in in the index.js could just have a better name.